### ISSUE TITLE
Add security headers

### ISSUE THREAD

**Issue by theborakompanioni** [joinmarket-webui/joinmarket-webui-docker#13]
Since we are in control of nginx we can add security headers, e.g.
```
        add_header X-Content-Type-Options "nosniff" always;
        add_header X-Frame-Options SAMEORIGIN always;
        add_header X-XSS-Protection "1; mode=block" always;
```
(just an example)

Add headers on demand (`Strict-Transport-Security`, `Content-Security-Policy`, etc.).

**Comment by Abhijay007** [comment]
Hi @theborakompanioni I've already created a PR for the cache headers, which is still pending review. In the meantime, would it be okay if I gave this a try as well?

**Comment by theborakompanioni** [comment]
> Hi [@theborakompanioni](https://github.com/theborakompanioni) I've already created a PR for the cache headers, which is still pending review. In the meantime, would it be okay if I gave this a try as well?

Yes, of course! I am not to familiar with this so it would be nice if you want to give it a shot!

**Comment by theborakompanioni** [comment]
Resolved with #152, #203 and #204
